modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GeneNameDescProjection_conf.pm

```diff
@@ -374,6 +376,60 @@ sub pipeline_analyses {
                             subject => $self->o('pipeline_name').' has completed',
                             text    => 'Log files: '.$self->o('output_dir'),
                           },
+      -flow_into => [ 'species_update_factory' ],
+    },
+    {
```

Comment:
This is almost exactly what is in Bio::EnsEMBL::Compara::PipeConfig::Parts::UpdateMemberNamesDescriptions - so rather than replicate (and maintain) it, could instead have: `@{Bio::EnsEMBL::Compara::PipeConfig::Parts::UpdateMemberNamesDescriptions::pipeline_analyses_member_names_descriptions($self)}`
with a default parameter value for `update_capacity`, and a resource_classes function that inherits from the parent and adds `1Gb_job`. (`compara_db` is set as a pipeline wide parameter, so doesn't need to be passed explicitly.)